server/status/feed.go

```diff
@@ -40,6 +42,13 @@ type CallErrorEvent struct {
 	Method proto.Method
 }
 
+// CanaryEvent is intended for testing only; it can be used by tests to
```

Comment:
I'm not sure I like the name Canary for this.  Canary is to test if a system is down, not to synchronize times or announce completion.